### DISCUSSION TITLE
Welcome to WorkX-feedback Discussions!

### DISCUSSION THREAD

**Discussion by webtrigon** [webtrigon/WorkX-feedback#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by navaneeth-webtribe** [comment]
### ROADMAP 🚀 💹 

**Product Feature Discussions:**

1. **Auto Checkout Functionality:**
   - If the user is offline and hasn’t checked out after 9 pm and is in Do Not Disturb (DND) mode, an automatic checkout will occur.
   - Notify the user about the auto-checkout and announce it in the timesheet channel. 🕒🛒

2. **User Notification for DND Mode:**
   - If the user is online and in DND mode, notify them to turn off DND if they are working.
   - Limit notifications to a maximum of 3 times within a 3-hour window or once per day. 📵🚨

3. **Delegating WFH or Leave Requests:**
   - Allow supervisors to delegate work-from-home (WFH) or leave requests to another supervisor for efficient management. 🔄🏡

4. **Django Admin Google SSO Login:**
   - Enable Google Single Sign-On (SSO) login in Django admin, supporting both SSO and traditional username-password logins. 🔐💻

**Improvement on Developer Experience:**

1. **Django Test Cases:**
   - Implement comprehensive test cases with a detailed report, including code and execution, before deploying to production. 🧪📊

2. **Readme.md Updates:**
   - Maintain updated readme.md files for each module to facilitate better understanding and documentation. 📝🔄

3. **Pylint/Black Formatter Setup:**
   - Set up Pylint and Black formatter to enhance code quality and maintain consistency. 🛠️📏

**Server/DevOps:**

1. **HTTP/3 Setup:**
   - Implement HTTP/3 on the server for improved performance and security. 🚀🔐

2. **Release Notes on Slack:**
   - Generate release notes based on commits and send them to Slack channels during production deployments. 📣🚀

3. **CI/CD Setup:**
   - Establish a robust Continuous Integration/Continuous Deployment (CI/CD) pipeline for efficient development and deployment processes. 🔄🚀

4. **Helm Charts for Kubernetes:**
   - Develop Helm charts to simplify the setup and deployment of the application in a Kubernetes environment. 📊🎛️
